README.md

```diff
@@ -2,20 +2,32 @@
 
 ## EYE1
 
-The legal possession of this `CRED` token is [recordat]
+The legal possession of this `CRED` token is [recordat]() of the _social engagement_, skills, and side-projects of the owner. In conjunction with srCRED [^1] ¿TBA?, it is LexDAO's assessment (passive and active/explicit voting) of participating in legal engineering activities including:
+- POAP attendance in studygroups or hackerthons;
+- TCR - indicating preference in token curated registries [^2] for an activity or priority;
+- funded activities such as working for membership or gaining a levelup via hackathon quest.
+
+## HOW2
+
 An outline of the current workflow to deploy a simple custom ERC 721 NFT Token
 
-# Plan Framework and Concept
------------------
-## Date Marker
-The plan frameowork and concept were last updated 2023-03-08
+Membership Flow Mockup -  [Excalidraw link](https://excalidraw.com/#room=2129b2214566ba2e246d,ykrk2lDXqfz4XnEUXArLBQ)
+  ![Membership Join Flow](https://user-images.githubusercontent.com/106759485/229314079-b935c399-103d-48dc-9dc1-b703409a6262.png)
 
+> [!CAUTION]
+> There will be progressive upgrade in the associated contracts ... currently by minting a new token every year, the information loses continuity so an important worktask is to find the right improvements in ERC721 such as handles and not pointers.
 
-## Core Script Files
-[LexDAO Token Generator](https://github.com/cimplylimited/file-processing-scripts)
+## WHO4
+Only for paidup members (opt-out) and l'externs (mandatory). Guests are assumed by default to be anonymous (but can voluntarily opt-in) and will not participate in the srCRED measurement (though their impact may flow onto members as in popular choice voting for most popular mentor).  
```

Comment:
That's fine.  We should have a concept of delegated authority discussed at length with pros and cons as a general philosophical consideration to overlay into aspects of the DAO.